zingolib/src/wallet/keys/unified.rs

```diff
@@ -803,6 +803,49 @@ impl TryFrom<&WalletCapability> for sapling_crypto::keys::OutgoingViewingKey {
     }
 }
 
+impl TryFrom<&WalletCapability> for UnifiedSpendingKey {
+    type Error = io::Error;
+
+    fn try_from(value: &WalletCapability) -> Result<Self, Self::Error> {
+        let transparent = &value.transparent;
+        let sapling = &value.sapling;
+        let orchard = &value.orchard;
+        match (transparent, sapling, orchard) {
+            (Capability::Spend(tkey), Capability::Spend(skey), Capability::Spend(okey)) => {
+                let mut key_bytes = Vec::new();
+                // orchard Era usk
+                key_bytes.write_u32::<LittleEndian>(BranchId::Nu5.into())?;
```

Comment:
Should the BranchId variant be hard-coded?

Reply:
As opposed to?